### ISSUE TITLE
Support encoder on Ubuntu & Tizen

### ISSUE THREAD

**Issue by baek2sm** [nnstreamer/nntrainer#2632]
The encoder-related part used in the current language model works well in Android, but not in Tizen, and sometimes or not depending on the environment in ubuntu (e.g., it does not work in our ubuntu environment of CI, and the reason why CI was passed in the past is that the corresponding code has never been operated in ubuntu of CI due to incorrect meson settings.).

The PR involved is #2627, and the codes involved are `#include “json.hpp”` and `#include<encoder.hpp>` of LLM examples.

In summary, `#include “json.hpp”` and `#include<encoder.hpp>` have issues that do not work properly in the ubuntu and tizen environments now.
